assets/styles/_colours.scss

```diff
@@ -3,6 +3,10 @@ $ho-brand: #8f23b3;
 $black: #0b0c0c;
 $white: #fff;
 $background: #f3f3f3;
+$grey-1: #6F777B;
```

Comment:
`govuk-colour('grey-1') `

replaced with `$grey-1` in all-components/ asl pages/projects